src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/OneVersusAllTrainer.cs

```diff
@@ -227,24 +222,25 @@ public override MulticlassPredictionTransformer<OneVersusAllModelParameters> Fit
                         var transformer = TrainOne(ch, Trainer, td, i);
                         featureColumn = transformer.FeatureColumnName;
                     }
+                    predictors[i] = (T)TrainOne(ch, Trainer, td, i).Model;
 
-                    predictors[i] = TrainOne(ch, Trainer, td, i).Model;
                 }
             }
 
-            return new MulticlassPredictionTransformer<OneVersusAllModelParameters>(Host, OneVersusAllModelParameters.Create(Host, _options.UseProbabilities, predictors), input.Schema, featureColumn, LabelColumn.Name);
+            return new MulticlassPredictionTransformer<OneVersusAllModelParameters<T>>(Host, OneVersusAllModelParameters<T>.Create(Host, _options.UseProbabilities, predictors), input.Schema, featureColumn, LabelColumn.Name);
         }
     }
 
     /// <summary>
-    /// Model parameters for <see cref="OneVersusAllTrainer"/>.
+    /// Model parameters for <see cref="OneVersusAllTrainer{T}"/>.
     /// </summary>
-    public sealed class OneVersusAllModelParameters :
+    public sealed class OneVersusAllModelParameters<T> :
```

Comment:
Is this change backward compatible? What if a user is using `OneVersusAllModelParameters` (no `<T>`)?

Reply:
It is not backwards compatible. After discussing the change with @codemzs we decided to leave this as a breaking change.

Reply:
@eerhardt, any comment?

Reply:
I've made a similar comment [here](https://github.com/dotnet/machinelearning/pull/3928#discussion_r299493172). I don't believe making an API breaking change on a minor version release is the right decision.